bad.js

```diff
@@ -0,0 +1,4 @@
+var bad = function(){
+  let foo = 1;
```

Comment:
'let' declarations in non-strict mode are not supported yet on Node 4.0.0. (node/no-unsupported-features)
'foo' is assigned a value but never used. (no-unused-vars)